jdaviz/app.py

```diff
@@ -422,7 +422,12 @@ def _link_new_data(self, reference_data=None, data_to_be_linked=None):
         # The glue-astronomy SpectralCoordinates currently seems incompatible with glue
         # WCSLink. This gets around it until there's an upstream fix.
         if isinstance(linked_data.coords, SpectralCoordinates):
-            wc_old = ref_data.world_component_ids[-1]
+            # Need the else case for mosviz.load_1d_spectra(), since
+            # the reference data will be 1D.
+            if len(ref_data.world_component_ids) > 1:
+                wc_old = ref_data.world_component_ids[-1]
+            else:
+                wc_old = ref_data.world_component_ids[0]
```

Comment:
the else case only covers `len(ref_data.world_component_ids) == 1`, right?  Why would `[0]` act differently than `[-1]` for a length of 1?

Reply:
Wondering the same thing 🤔